lmfit/lineshapes.py

```diff
@@ -412,6 +413,40 @@ def thermal_distribution(x, amplitude=1.0, center=0.0, kt=1.0, form='bose'):
     return real(1/(amplitude*exp((x - center)/not_zero(kt)) + offset + tiny*1j))
 
 
+def bose(x, amplitude=1.0, center=0.0, kt=1.0):
+    """Return a Bose-Einstein thermal distribution function.
+
+    bose(x, amplitude=1.0, center=0.0, kt=1.0) =  amplitude/(exp((x - center)/kt) - 1)
+
+    Notes
+    -----
+    - `kt` should be defined in the same units as `x`. The Boltzmann
+      constant is ``kB = 8.617e-5 eV/K``.
+
+    See Also
+    --------
+    thermal_distribution, boltzmann, fermi
+    """
+    return amplitude/(exp((x - center)/not_zero(kt)) - 1)
+
+
+def fermi(x, amplitude=1.0, center=0.0, kt=1.0):
+    """Return a Fermi-Dirac thermal distribution function.
+
+    fermi(x, amplitude=1.0, center=0.0, kt=1.0) =  amplitude/(exp((x - center)/kt) -+ 1)
+
+    Notes
+    -----
+    - `kt` should be defined in the same units as `x`. The Boltzmann
+      constant is ``kB = 8.617e-5 eV/K``.
+
+    See Also
+    --------
+    thermal_distribution, boltzmann, bose
```

Comment:
you removed the `boltzman` function again, so that should be removed here as well